adcircpy/forcing/winds/best_track.py

```diff
@@ -275,6 +284,15 @@ def make_request():
 
         self.__atcf = io.BytesIO(response.read())
 
+    @property
+    def external_track(self) -> str:
+        return self.synthetic_file_name
+
+    @external_track.setter
+    def external_track(self, file_with_storm: str):
+        self.__atcf = io.open(file_with_storm, "rb")
+        self.synthetic_file_name = file_with_storm
```

Comment:
`self.synthetic_file_name` should be `self.__external_track_file_name` instead IMO?

Reply:
> `self.synthetic_file_name` should be `self.__external_track_file_name` instead IMO?

Yes, but we should just probably use single underscore from now on: `self._external_track`.

Reply:
I'm a little confused how this convention works as it can easily lead to recursion where the private method calls the public method. I've renamed the field to `external_file_name` 

I think the visibility of this field should be public as the user could introspect it for some kind of logic in a script.